app/views/users/show.html.erb

```diff
@@ -4,25 +4,26 @@
   <h1><%= @user.name %></h1>
 </div>
 
-
-<div class="panel panel-default">
-  <div class="panel-heading">
-    <div class="row">
-      <div class="col-md-6">
-        <h4><%= t('.user_data') %></h4>
+<% if !@user.is_superadmin? %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <div class="row">
+          <div class="col-md-6">
+            <h4><%= t('.user_data') %></h4>
+          </div>
+          <div class="col-sm-6 text text-right">
+            <%= link_to t('.edit', default: t('helpers.links.edit')), edit_user_path, class: 'btn btn-primary' if @user == current_user %>
+          </div>
+        </div>
       </div>
-      <div class="col-sm-6 text text-right">
-        <%= link_to t('.edit', default: t('helpers.links.edit')), edit_user_path, class: 'btn btn-primary' if @user == current_user %>
+      <div class="panel panel-body">
+        <%= render 'public_user_data' %><br>
+        <%= render 'private_user_data' if @user == current_user %>
       </div>
     </div>
-  </div>
-  <div class="panel panel-body">
-    <%= render 'public_user_data' %><br>
-    <%= render 'private_user_data' if @user == current_user %>
-  </div>
-</div>
+<% end %>
 
-<% if @user == current_user %>
+<% if !@user.is_superadmin? && @user == current_user %>
```

Comment:
An der Stelle jetzt noch den Block in den oberen schieben? Dann kann man sich ein 
if !@user.is_superadmin?
sparen